web/templates/policy/copyright.html.md

```diff
@@ -0,0 +1,102 @@
+and their creations may be depended on by thousands of people and companies.
+
+Users may submit counter notices to your copyright complaint (see below for more
+details). You will be notified of the counter notice, we will re-enable their
+content within 14 business days if you do not inform us that you have initiated legal
+action to restrain the user from engaging in copyright infringing activity on Hex.
+
+Please note that all submissions will be published to the [Lumen Database](https://lumendatabase.org/)
+with any personal information removed.
+
+#### Your complaint must follow these rules:
+
+1. Identify the copyrighted work you believe has been infringed.
+
+2. Identify the material that you allege is infringing the copyrighted work.
+
+3. Explain what the affected user would need to do in order to remedy the infringement
+
+4. Provide your contact information.
+
+5. Provide contact information, if you know it, for the alleged infringer.
+
+6. Include the following statement: "I have read and understand Hex's Copyright Policy."
+
+7. Include the following statement: "I have a good faith belief that use of the
+   copyrighted materials described above on the infringing web pages is not authorized
+   by the copyright owner, or its agent, or the law."
+
+8. Also include the following statement: "I swear, under penalty of perjury, that
+   the information in this notification is accurate and that I am the copyright
+   owner, or am authorized to act on behalf of the owner, of an exclusive right
+   that is allegedly infringed."
+
+9. Include your physical or electronic signature.
+
+Submit your copyright notice to [copyright@hex.pm](mailto:copyright@hex.pm). You
+may include attachments but please include a copy of your letter in plain-text
+in the body of the email.
+
+### How to submit a DMCA takedown counter notice
+
+The DMCA requires that you swear to the facts in your notice under penalty of
+perjury. Under US law it's a federal crime to give false information under oath.
+
+Please conduct a thorough investigation, probably with the help of legal counsel,
+before submitting a takedown counter notice to ensure that it is valid. Submitting
+a DMCA takedown counter notice can have real legal consequence, the other party
+may start legal action against you to ensure your content is disabled.
+
+In order to file a counter notice, you must have "a good faith belief that the
+material was removed or disabled as a result of mistake or misidentification of
+the material to be removed or disabled." Examples of mistakes in the takedown
+notice could be that your content falls under fair-use or you may have a license
+for the copyrighted content. It is up to you if you want to explain why the DMCA
+takedown notice is not valid.
+
+Please note that all submissions will be published to the [Lumen Database](https://lumendatabase.org/)
+with any personal information removed.
```

Comment:
This and the other dupe are under their own sections, submitting a takedown notice and submitting a counter notice, respectively. Maybe they should be moved out to a location where they don't have to be duped?